.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: android
+jdk: oraclejdk8
+dist: trusty
+
+env:
+  global:
+    - ANDROID_API_LEVEL=28
+    - ANDROID_BUILD_TOOLS_VERSION=28.0.3
+
+android:
+  licenses:
+    - 'android-sdk-preview-license-.+'
+    - 'android-sdk-license-.+'
+    - 'google-gdk-license-.+'
+  components:
+    - tools
+    - platform-tools
+    - build-tools-$ANDROID_BUILD_TOOLS_VERSION
+    - android-$ANDROID_API_LEVEL
+
+
+before_install:
+  - yes | sdkmanager "platforms;android-$ANDROID_API_LEVEL"
+
+script:
+  - ./gradlew build check
```

Comment:
shouldn't this include ./build.sh

Reply:
Are you referring to a build.sh that already exists or are you suggesting we should write a build.sh that bundles all our build commands?